lisa/runners/lisa_runner.py

```diff
@@ -898,3 +900,17 @@ def _create_platform_requirement(self) -> Optional[schema.NodeSpace]:
             )
 
         return platform_requirement
+
+    def _copy_env_log(self, environment: Environment, test_result: TestResult) -> None:
+        if is_unittest():
```

Comment:
@squirrelsc is this fine? Unit test does not have environment folder. The value is ".", so this code would attempt to copy entire lisa code under the test log folder for unittest

Reply:
Yes, it's ok. But write your comments in code, please.

Reply:
updated

Reply:
Please add below too. :)

`The value is ".", so this code would attempt to copy entire lisa code to the test log folder for unittest`